docs/src/main/asciidoc/quartz.adoc

```diff
@@ -400,6 +400,31 @@ After a few seconds, in another terminal, run `curl localhost:8080/tasks` to ver
 
 You can also generate the native executable with `./mvnw clean package -Pnative`.
 
+[[quartz-register-plugin-listeners]]
+== Registering Plugin and Listeners
+
+You can register a plugin, jobListener or triggerListener through Quarkus configuration.
+
+The example bellow register the plugin `org.quartz.plugins.history.LoggingJobHistoryPlugin` named as `jobHistory` with the property `jobSuccessMessage` defined as `Job [{1}.{0}] execution complete and reports: {8}` 
+
+[source,conf]
+----
+quarkus.quartz.plugin.jobHistory.class=org.quartz.plugins.history.LoggingJobHistoryPlugin
+quarkus.quartz.plugin.jobHistory.jobSuccessMessage=Job [{1}.{0}] execution complete and reports: {8}
+----
+
+You can also register a jobListener or a triggerListener through Quartz API by injecting the underlying `org.quartz.Scheduler`
```

Comment:
You can also register a listener programmatically with an injected `org.quartz.Scheduler`...

Reply:
Done! Really seems better this way!

Reply:
Seems something is wrong with the CI itself.
The previous run was successful and on this one, I've only changed the docs.

/cc @machi1990

Reply:
It could be only a case of failing random test. Can you rebase on master and then push again to re-launch CI?

Reply:
I'll do that. Thanks!